functional-tests/test-bed/src/main/java/org/arquillian/smart/testing/ftest/testbed/project/ProjectBuilder.java

```diff
@@ -70,6 +70,8 @@ private BuiltProject executeGoals(String... goals) {
                     .setProperties(asProperties(buildConfigurator.getSystemProperties()))
                     .skipTests(buildConfigurator.isSkipTestsEnabled())
                     .setMavenOpts(buildConfigurator.getMavenOpts())
+                    .setWorkingDirectory(buildConfigurator.getWorkingDirectory())
+                    .setAlsoMake(true)
```

Comment:
otherwise it'll install all dependent projects even if it's present on your machine which will increase build time.

e.g. In case of one sub-module is dependent on other sub-modules. I already have this submodules installed in my `~/.m2` which is my local, but still it will install it again as we set `also-make` always true which is not necessary here, but required for `CI` environment